test/chi/custom-elements/buttons.pug

```diff
@@ -170,3 +171,20 @@ each size in ['default', 'sm', 'md', 'lg', 'xl']
         td
           chi-button(type='close', size= size !== 'default' ? size : '')
 
+script.
+  function getElementHeight(element) {
+    return parseInt(window.getComputedStyle(element, null).getPropertyValue("height").replace("px", ""));
+  }
+
+  function checkButtonHeight() {
```

Comment:
I'll change the name of the function just to make it more understandable about what it actually is meant to do. Like, for example: allButtonsRendered. @janibolkvadze what do you think?

Reply:
Good point, thanks, I changed the function name to **checkAllButtonsRendered**